test/quantization/test_quantize_jit.py

```diff
@@ -1038,6 +1038,19 @@ def forward(self, x):
                    .check("quantized::linear") \
                    .run(model.graph)
 
+    def test_inplace_option(self):
+        for tracing in [True, False]:
+            model = get_script_module(torch.nn.Conv2d(3, 3, 3), tracing, self.img_data[0][0])
```

Comment:
I don't have too much context, but do we need to test (not in this PR, in the future) inplace on real world models to check the long tail of corner cases?  Or is this sufficient to prove that it works.

Reply:
I don't think we'll need to test `inplace` for all models, most of the models should just work. Although I'm not 100% sure if there will be cases that inplace does not work, maybe if we uncover such cases we can fix the problem add them to the test